SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialStatics.h

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Kismet/BlueprintFunctionLibrary.h"
+#include "SpatialStatics.generated.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogSpatialStatics, Log, All);
+
+UCLASS()
+class SPATIALGDK_API USpatialStatics : public UBlueprintFunctionLibrary
+{
+	GENERATED_BODY()
+
```

Comment:
Responding to my earlier comment somewhere that can be responded to or resolved :P
Re - do we think this is what this is going to look like : I mean do we think these functions are staying around.
On further reflection we can deprecate these if they are not without much effort. So it doesn't really matter.